qp_klp/process_amplicon_job.py

```diff
@@ -290,6 +291,8 @@ def process_amplicon(mapping_file_path, qclient, run_identifier, out_dir,
                         data_type = key
                         break
 
+                # add_sif_info.append(metadata['sample_name', 'project_name'])
```

Comment:
needed?

Reply:
Nope. Originally I didn't think I needed to query Qiita for a list of BLANKs, so I had an alternate method. I kept it in as a point of discussion. I'll remove it.